### PR TITLE
Update json-and-xml-views.rst

### PR DESCRIPTION
**Pull request by ConAfectoManu**
Hello.
I am a Spanish speaker, I have been working for a long time with CakePHP and I want to contribute to those who begin to work with this development framework, so I have started trying to translate this article and my intention is to continue doing it with others.
Greetings.

### CODE REVIEW

File: es/views/json-and-xml-views.rst [hunk 1]
```diff
@@ -1,11 +1,254 @@
...
+======================
+
+.. php:class:: XmlView
+
+De forma predeterminada, cuando se utiliza ``serialize``, XmlView ajustará
+las variables de vista serializadas con un nodo ``<response>``. Puede
+establecer un nombre personalizado para este nodo mediante la opción
+``rootNode``.
+
+La clase XmlView admite la opción``xmlOptions`` que le permite personalizar
```
Comment (othercorey) [inline es/views/json-and-xml-views.rst]:
```suggestion
La clase XmlView admite la opción ``xmlOptions`` que le permite personalizar
```

File: es/views/json-and-xml-views.rst [hunk 2]
```diff
@@ -1,11 +1,254 @@
...
+
+De forma predeterminada, cuando no se habilitan las :ref:`file-extensions`, se
+utiliza la solicitud, seleccionando el encabezado ``Accept``, seleccionando qué
+tipo de formato se debe presentar al usuario. Un ejemplo de formato ``Accept``
+que se utiliza para representar respuestas ``JSON`` es ``application/json``.
+
+Uso de vistas de datos con la clave Serialize
+=============================================
+
+La opción ``serialize'' indica qué variable(s) de vista se deben serializar
```
Comment (othercorey) [inline es/views/json-and-xml-views.rst]:
```suggestion
La opción ``serialize`` indica qué variable(s) de vista se deben serializar
```

File: es/views/json-and-xml-views.rst [hunk 3]
```diff
@@ -1,11 +1,254 @@
...
+personalizado en lugar de "callback", establezca ``_jsonp`` al nombre requerido en
+lugar de ``true.``.
+
+Ejemplo de uso
+==============
+
+Si bien el :doc:`RequestHandlerComponent 
+</controllers/components/request-handling>` puede establecer automáticamente la 
+vista en función del tipo de contenido o la extensión de la solicitud, también puede
+controlar las asignaciones de vistas en el controlador:::
```
Comment (othercorey) [inline es/views/json-and-xml-views.rst]:
```suggestion
controlar las asignaciones de vistas en el controlador::
```

**Comment by othercorey** [comment]
@markstory the build fails

**Review by ConAfectoManu — COMMENTED** [review]
All in all right. Thanks.

**Pull request merged**